lib/util/ImpCentralApiHelper.js

```diff
@@ -320,25 +322,23 @@ class ImpCentralApiHelper {
     }
 
     createEntity(entityType, ...args) {
-        const isLoginKey = entityType === Identifier.ENTITY_TYPE.TYPE_LOGIN_KEY;
         const entityApi = this._getEntityApi(entityType);
         return this._checkAccessToken().
             then(() => this._entityStorage.clearList(entityType)).
             then(() => this._processImpCentralApiRequest(
-                !isLoginKey, null, null, entityApi.create.bind(entityApi), ...args)).
+                true, null, null, entityApi.create.bind(entityApi), ...args)).
```

Comment:
Do we still have use cases where the first parameter is false?

Reply:
Yes, for auth requests like login with user/pwd, refresh access token, etc